### DISCUSSION TITLE
Font-awesome question

### DISCUSSION THREAD

**Discussion by Kathou99** [Lissy93/dashy#659] [Help and Support] [answered]
Hi, Does is all the font awesome I can use?

Because Some work and some don't

It is all free font awesome.

Like one I had try but no succes only a empty box

Thank :) 

<img width="41" alt="dashy here" src="https://user-images.githubusercontent.com/71993497/168454509-18f2d750-8212-4076-93ef-d7ae210c1dfc.PNG">
<img width="186" alt="ddashy 2" src="https://user-images.githubusercontent.com/71993497/168454512-810f792c-d69a-4092-9122-119d8d96941a.PNG">

**Comment by lissy93** [comment, marked as answer]
Yeah, they should all work.
But looks like you've got a small typo in that example. Raspberry Pi is a brand, so the icon should be: `fab fa-raspberry-pi` :)

---

The icon style category (`fas`, `far`, `fal`, `fad` and `fab`) is something set by Font Awesome. So although it seems annoying, there's nothing that can be done on Dashy's side to improve it. If it helps, with the exception of brand icons (`fab`), you'll usually want all your icons to be the same category so they look uniform, which makes things a bit easier.

**Comment by kindomLee** [comment]
I have also encountered similar problems
"far fa-tree-deciduous"
As long as the name contains more than one dash icon, it will not be displayed properly

**Reply by erohtar** [reply]
I'm facing this too

**Reply by lissy93** [reply]
Can you share the error message in your browser console ([here's how](https://github.com/Lissy93/dashy/blob/master/docs/troubleshooting.md#how-to-open-browser-console)) + the relevant parts of your config.

If you've followed the docs, and checked [this troubleshooting item](https://github.com/Lissy93/dashy/blob/master/docs/troubleshooting.md#font-awesome-icons-not-displaying), there shouldn't be any issues on Dashy's side.

**Reply by erohtar** [reply]
Everything else being the same..
This works: far fa-wave-square
This doesn't: far fa-heart-pulse

No error in the browser console:
![image](https://user-images.githubusercontent.com/5120628/190924579-c1e44376-5d56-4942-9599-11ec7444887a.png)

**Reply by erohtar** [reply]
** This doesn't: fas fa-heart-pulse
(the regular icon is pro)

**Reply by lissy93** [reply]
Ah, that's because only [Font Awesome V5](https://fontawesome.com/v5/search) works at the moment.

**Reply by erohtar** [reply]
Ohhh.. got it! A little bit of a bummer though lol

**Reply by lissy93** [reply]
Yeah, sorry about that. I will update it to V6 in the soon, but I need to find a way to keep the old icons backwards compatible, so that I don't break everyone's existing setup.

**Reply by erohtar** [reply]
No worries -  glad it's on your list

**Comment by chevdor** [comment]
A live preview when editing would be convenient.

**Reply by lissy93** [reply]
See https://github.com/Lissy93/dashy/issues/749
